polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -447,6 +469,7 @@ def move_to_ee_pose(
         delta: bool = False,
         Kx: torch.Tensor = None,
         Kxd: torch.Tensor = None,
+        op_space_interp: bool = True,
         **kwargs,
     ) -> List[RobotState]:
         """Uses an operational space controller to move to a desired end-effector position (and, optionally orientation).
```

Comment:
Documentation here would be useful: what does op_space_interp do, as well as the fact that we're actually not doing opspace control anymore.